src/main/java/com/aws/greengrass/util/platforms/WindowsPlatform.java

```diff
@@ -153,4 +156,27 @@ public ShellDecorator withShell(String shell) {
             throw new UnsupportedOperationException("changing shell is not supported");
         }
     }
+
+    @Override
+    public String prepareDomainSocketFilepath(Path rootPath) {
+        return NAMED_PIPE;
+    }
+
+    @Override
+    public String prepareDomainSocketFilepathForComponent(Path rootPath) {
```

Comment:
Change this name? It has domainsocket in the name.

Reply:
Yeah, I probably should rename it in the first place. Would something with "IPC" make sense? For example, this method would call `prepareIpcFilepathForComponent`.

Reply:
Updated